packages/gafl-webapp-service/src/pages/renewals/renewal-inactive/__tests__/route.spec.js

```diff
@@ -1,45 +1,236 @@
+        'renewal_inactive_not_due_1 abc-123 renewal_inactive_has_expired_1 12th Never renewal_inactive_has_expired_2',
+        { renewal_inactive_title_2: 'Renewal inactive title 2' },
+        'Renewal inactive title 2'
+      ],
+      [
+        RENEWAL_ERROR_REASON.NOT_ANNUAL,
+        'renewal_inactive_not_due_1 abc-123 renewal_inactive_not_annual_1',
+        { renewal_inactive_title_3: 'Renewal inactive title 3' },
+        'Renewal inactive title 3'
+      ],
+      ['', '', {}, '']
+    ])('title and body message', (authReason, expectedBodyString, mssgOverrides, expectedTitle) => {
+      const getMessages = (overrides = {}) => ({
+        renewal_inactive_not_due_1: 'renewal_inactive_not_due_1 ',
+        renewal_inactive_not_due_2: ' renewal_inactive_not_due_2 ',
+        renewal_inactive_has_expired_1: ' renewal_inactive_has_expired_1 ',
+        renewal_inactive_has_expired_2: ' renewal_inactive_has_expired_2',
+        renewal_inactive_not_annual_1: ' renewal_inactive_not_annual_1',
+        renewal_inactive_title_1: 'renewal_inactive_title_1',
+        renewal_inactive_title_2: 'renewal_inactive_title_2',
+        renewal_inactive_title_3: 'renewal_inactive_title_3',
+        ...overrides
+      })
 
-  const mockRequest = {
-    cache: () => ({
-      helpers: {
-        status: {
-          getCurrentPermission: mockStatusCacheGet
+      const getMockRequest = (authReason, messages = getMessages()) => ({
+        cache: () => ({
+          helpers: {
+            status: {
+              getCurrentPermission: async () => ({
+                referenceNumber: 'abc-123',
+                authentication: {
+                  endDate: '',
+                  reason: authReason
+                }
+              })
+            }
+          }
+        }),
+        i18n: {
+          getCatalog: () => messages
+        },
+        locale: 'en-gb'
+      })
+
+      it('body message is as expected', async () => {
+        const mockRequest = getMockRequest(authReason)
+        moment.mockImplementation(
+          getMomentMockImpl({
+            format: () => '12th Never'
+          })
+        )
+        const { bodyMessage } = await getData(mockRequest)
+        expect(bodyMessage).toEqual(expectedBodyString)
+      })
+
+      it('title is as expected', async () => {
+        const mockRequest = getMockRequest(authReason, getMessages(mssgOverrides))
+        moment.mockImplementation(getMomentMockImpl())
+        const { title } = await getData(mockRequest)
+        expect(title).toEqual(expectedTitle)
+      })
+    })
+
+    const mockStatusCacheGet = jest.fn()
+
+    const mockRequest = {
+      cache: () => ({
+        helpers: {
+          status: {
+            getCurrentPermission: mockStatusCacheGet
+          }
         }
-      }
-    }),
-    i18n: {
-      getCatalog: jest.fn(() => englishTranslations)
+      }),
+      i18n: {
+        getCatalog: jest.fn(() => englishTranslations)
+      },
+      locale: 'en-gb'
```

Comment:
This is a bit messy - there's already a function to get a mock request, this should be rolled into it. Also, we shouldn't be importing English and Welsh translations, these should be mocked otherwise there's a danger of tight coupling.